src/index.js

```diff
@@ -1005,15 +999,31 @@ class Enmap extends Map {
     }
   }
 
-  /*
-   * INTERNAL method to verify the type of a key or property
+  /** 
+   * INTERNAL METHOD. Checks if the key is valid.
+   * Will THROW AN ERROR if the key is nil or not a string.
+   * @param {string|Array<string>} keys The key(s) to check.
+   */
+  #checkKeyOrArrayOfKeys(keys) {
+    if (!(typeof keys === 'string' || (Array.isArray(keys) && keys.every(e => typeof e === "string")))) {
```

Comment:
I don't believe a decent chunk of the methods using this support an array of keys.

Reply:
Yes but the two extra checks shouldn't hurt.

Reply:
Well it would cause it you pass an array of strings it now will accept it

Reply:
Oh I am dumb. You're right... That's a problem... One moment